seastar.yaml

```diff
@@ -0,0 +1,243 @@
+package:
+  name: seastar
+  epoch: 0
+  version: "25.05.0"
+  description: "Seastar is a high-performance, open-source framework for building C++ applications"
+  copyright:
+    - license: "Apache-2.0"
+  dependencies:
+    runtime:
+      - binutils
+      - boost-dev
+      - c-ares-dev
+      - fmt-dev
+      - gcc
+      - pkgconf
+      - protobuf-dev
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp-dev
+      - bison
+      - boost-dev
+      - build-base
+      - c-ares-dev
+      - doxygen
+      - fmt-dev
+      - gnutls-dev
+      - gtest-dev
+      - hwloc-dev
+      - libsctp-dev
+      - liburing-dev
+      - lz4-dev
+      - numactl-dev
+      - openssl-dev
+      - pkgconf-dev
+      - protobuf-dev
+      - python-${{vars.py-version}}
+      - ragel
+      - util-linux-dev
+      - valgrind-dev
+      - xfsprogs-dev
+      - yaml-cpp-dev
+
+vars:
+  py-version: 3.13
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: "https://github.com/scylladb/seastar.git"
```

Comment:
Seastar for Redpanda has a public repository here: https://github.com/redpanda-data/seastar

Could we package `redpanda-seastar` instead?

Reply:
@EyeCantCU It doesn't look like they cut new releases but continue to work on branches and merge into master. We ok with building master on this repo? 
Also should this be in enterprise against wolfi?
Finally, do we still package the other open source seastar as just seastar?